### ISSUE TITLE
Instance download results in HTTP 400 errors

### ISSUE THREAD

**Issue by oxyCabhru** [killpowa/Twickt-Launcher#1530]
- [x] I searched the issue tracker for other issues covering my case
- [x] This is a Launcher issue. (Usually if you can reproduce it with a fresh instance)

**Describe the bug**
Downloading any new instance results in downloading game files very slowly.

**To Reproduce**
1. Open GDLauncher
2. Create new instance
3. Stuck at "Downloading game files..."
4. Console reads:
```
Error while downloading <http://resources.download.minecraft.net/bd/[...] | http://resources.download.minecraft.net/bd/[...]> to </home/oxy/.config/gdlauncher_next/datastore/assets/objects/bd/[...]> --> Request failed with status code 400
```

**Screenshots**
![Screenshot from 2023-01-28 17-13-58](https://user-images.githubusercontent.com/68921536/215274348-9f505b58-5d20-4bea-8285-a07a7ab002a7.png)

**Operating System:**
- OS: Fedora Workstation 37
- Launcher Version: 1.1.29
- Java Version: Auto downloaded
- Minecraft Version: 1.19.2
- Mod Loader: Forge

**Additional context**
Seems to have been solved already by #1514 , but the request uri evidently still points to http.

**Comment by Eskaan** [comment]
Duplicate of #1513

**Comment by Eskaan** [comment]
As I see you already found the corresponding PR, let me tell you: It is already in the released v1.1.30-beta.1, I plan on adding a few things and releasing a Stable this weekend.

**Comment by oxyCabhru** [comment]
Thanks for the quick reply.
